chromium_src/chrome/browser/ui/webui/settings/site_settings_helper.cc

```diff
@@ -95,25 +95,23 @@ std::string_view ContentSettingsTypeToGroupName(ContentSettingsType type) {
   return ContentSettingsTypeToGroupName_ChromiumImpl(type);
 }
 
-const std::vector<ContentSettingsType>& GetVisiblePermissionCategories() {
-  static base::NoDestructor<std::vector<ContentSettingsType>> types{
-      {ContentSettingsType::AUTOPLAY, ContentSettingsType::BRAVE_ETHEREUM,
-       ContentSettingsType::BRAVE_SOLANA,
-       ContentSettingsType::BRAVE_GOOGLE_SIGN_IN,
-       ContentSettingsType::BRAVE_LOCALHOST_ACCESS}};
-  static bool initialized = false;
-  if (!initialized) {
-    auto& base_types = GetVisiblePermissionCategories_ChromiumImpl();
-    types->insert(types->end(), base_types.begin(), base_types.end());
-
-    if (!base::FeatureList::IsEnabled(blink::features::kBraveWebSerialAPI)) {
-      std::erase(*types, ContentSettingsType::SERIAL_GUARD);
-    }
-
-    initialized = true;
+std::vector<ContentSettingsType> GetVisiblePermissionCategories(
+    const std::string& origin,
+    Profile* profile) {
+  static constexpr ContentSettingsType extra_types[] = {
+      ContentSettingsType::AUTOPLAY, ContentSettingsType::BRAVE_ETHEREUM,
+      ContentSettingsType::BRAVE_SOLANA,
+      ContentSettingsType::BRAVE_GOOGLE_SIGN_IN,
+      ContentSettingsType::BRAVE_LOCALHOST_ACCESS};
```

Comment:
pls add trailing comma.

Reply:
Fixed in f35be2835bd80a36b72cc1b210dfcffae55f1c83.